### PR TITLE
 [MIG] website_sale_hide_empty_category: Migration to 11.0 

### PR DESCRIPTION
**Pull request by remytms**
This PR upgrade the `website_sale_hide_empty_category` from 10.0 to 11.0 and move it from `website` to `e-commerce`.

### CODE REVIEW

**Comment by remytms** [comment]
Did I use the right process ? I have cherry-pick all the commit from this PR : https://github.com/OCA/website/pull/530

**Comment by pedrobaeza** [comment]
Yes! Thank you very much

**Review by pedrobaeza — APPROVED** [review]

File: website_sale_hide_empty_category/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2017 LasLabs Inc. <https://laslabs.com/>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+{
+    "name": "Website Sale - Hide Empty Categories",
+    "summary": "Hide any Product Categories that are empty",
+    "version": "11.0.1.0.0",
+    "category": "Website",
+    "website": "https://github.com/OCA/website/",
```
Comment (simahawk) [inline website_sale_hide_empty_category/__manifest__.py]: https://github.com/OCA/e-commerce

File: website_sale_hide_empty_category/README.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/license-LGPL--3-blue.svg
```
Comment (simahawk) [inline website_sale_hide_empty_category/README.rst]: png

**Review by simahawk — APPROVED** [review]
2 minor issues, approving to not get back ;)

**Comment by remytms** [comment]
@simahawk , @pedrobaeza , Thanks for yours reviews. Things are good now. :)

**Review by simahawk — APPROVED** [review]

**Comment by remytms** [comment]
@simahawk , Is it possible to merge this PR or do I need other review before that ? I think I can't do it  because I don't see the merge button.

**Comment by simahawk** [comment]
@pedrobaeza ping :)

**Comment by remytms** [comment]
@pedrobaeza , @simahawk , is it time to merge ? :D

**Comment by remytms** [comment]
@pedrobaeza Thanks a lot ! :D

**Pull request merged**